x/oracle/keeper/params.go

```diff
@@ -45,6 +45,19 @@ func (k Keeper) SetAcceptList(ctx sdk.Context, acceptList types.DenomList) {
 	k.paramSpace.Set(ctx, types.KeyAcceptList, acceptList)
 }
 
+// MandatoryList returns the denom list that are mandatory
+func (k Keeper) MandatoryList(ctx sdk.Context) (res types.DenomList) {
+	k.paramSpace.Get(ctx, types.KeyMandatoryList, &res)
+	return
+}
+
+// SetMandatoryList updates the mandatory list of assets supported by the x/oracle
+// module.
+func (k Keeper) SetMandatoryList(ctx sdk.Context,
+	mandatoryList types.DenomList) {
+	k.paramSpace.Set(ctx, types.KeyMandatoryList, mandatoryList)
+}
```

Comment:
I think [param proposal handler](https://github.com/cosmos/cosmos-sdk/blob/main/x/params/proposal_handler.go#L37) makes changes according to key and value pair directly. either we can make a custom param proposal handler for oracle that does it. Otherwise, the new proposal would have to reflect the changes for both mandatory list and accept list

Reply:
Makes sense. Wdyt about updating to the new generic governance proposal process available in 0.46?

Reply:
true, But i think the current flow makes a bit of sense, essentially adding a denom would be 2 step process 

1. add denom to accepted list to see if there are any issues to post price feed
2. add denom to mandatory list 

rather than adding a denom directly to mandatory and accepted list , would have to make sure that subset property is always maintained during the governance phase

Reply:
We should codify that instead of it being something necessary to remember, especially since we expect other parties to participate in governance

Reply:
I'll add one under the oracle V1 epic and we'll tackle it there